Zenodo.js

```diff
@@ -187,15 +188,7 @@ function scrape(doc, url) {
 			item.url = url;
 			if (abstract) item.abstractNote = abstract;
 
-			//Zenodo uses some non-existent CSL types we're fixing.
-			var zoteroType = {
-				"figure": "artwork",
-	 			"article": "report"
-			};
-
-			if (item.itemType == "document" && zoteroType[type]) {
-				item.itemType = zoteroType[type];
-			}
```

Comment:
Can we delete the first introduction of the variable `type` as well?

Reply:
Do you mean [L42](https://github.com/katrinleinweber/translators/blob/2b436bebeaf2af72c2c95cf20f5f21740ef2953b/Zenodo.js#L42) or [L133](https://github.com/katrinleinweber/translators/blob/2b436bebeaf2af72c2c95cf20f5f21740ef2953b/Zenodo.js#L133). The latter can indeed be removed without breaking the tests.

Reply:
Yes, the latter one seems not necessarily anymore. We certainly still need the first one.